src/sentry/conf/server.py

```diff
@@ -464,6 +465,7 @@ def env(key, default='', type=None):
 CELERY_QUEUES = [
     Queue('alerts', routing_key='alerts'),
     Queue('auth', routing_key='auth'),
+    Queue('commits', routing_key='commits'),
```

Comment:
@JTCunning these is going to get picked up by a `worker-glob` worker, right?

We also might need to explicitly add the queue to federation. I don't fully remember how that works. (Should prob document)